rules/docker-sonic-mgmt-framework.mk

```diff
@@ -0,0 +1,25 @@
+# docker image for mgmt-framework
+
+DOCKER_MGMT-FRAMEWORK_STEM = docker-sonic-mgmt-framework
+DOCKER_MGMT-FRAMEWORK = $(DOCKER_MGMT-FRAMEWORK_STEM).gz
+DOCKER_MGMT-FRAMEWORK_DBG = $(DOCKER_MGMT-FRAMEWORK_STEM)-$(DBG_IMAGE_MARK).gz
+
+$(DOCKER_MGMT-FRAMEWORK)_PATH = $(DOCKERS_PATH)/$(DOCKER_MGMT-FRAMEWORK_STEM)
+
+$(DOCKER_MGMT-FRAMEWORK)_DEPENDS += $(REDIS_TOOLS) $(SONIC_MGMT-FRAMEWORK)
+$(DOCKER_MGMT-FRAMEWORK)_DBG_DEPENDS = $($(DOCKER_CONFIG_ENGINE_STRETCH)_DBG_DEPENDS)
+
```

Comment:
Add _DBG_DEPENDS

Reply:
Thanks for the comment. Added below line
$(DOCKER_MGMT_FRAMEWORK)_DBG_DEPENDS += $(SONIC_MGMT_FRAMEWORK)

Reply:
DBG_DEPENDS is for packages with debug symbols. Not for the main package.

e.g. docker-orchagent.mk:$(DOCKER_ORCHAGENT)_DBG_DEPENDS +=   $(SWSS_DBG) \

Reply:
Can we standardize to use '_' instead of '-' ?

Reply:
Done. Changed to '_'

Reply:
Also changed below line
$(DOCKER_MGMT_FRAMEWORK)_DBG_DEPENDS += $(REDIS_TOOLS) $(SONIC_MGMT_FRAMEWORK_DBG)